docs/send-data/hosted-collectors/microsoft-source/azure-metrics-source.md

```diff
@@ -0,0 +1,99 @@
+
+<img src={useBaseUrl('img/send-data/azure-event-hub.svg')} alt="icon" width="40"/>
+
+Azure Metrics Source is used to collect all available metrics from Azure Monitor into Sumo Logic. To obtain a complete list of metrics that are collected using this source, refer to the [Azure Documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/metrics-index#supported-metrics-and-log-categories-by-resource-type).
+
+## Setup
+
+### Vendor configuration
+
+The Azure Metrics Source requires you to provide **Tenant Id**, **Client Id**, and **Client Secret** while configuring. To obtain these values, follow the below steps:
+
+1. Sign in to the [Azure portal](https://portal.azure.com/#create/Microsoft.Template/uri/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FSumoLogic%2Fsumologic-azure-serverless%2Fmain%2FAzureMetrics%2FRoleAssignmentSubscriptions.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FSumoLogic%2Fsumologic-azure-serverless%2Fmain%2FAzureMetrics%2FCreateUIDefSubscriptions.json).
+1. In the **Custom Deployment** page, under **Basics** tab, select the **Subscription**, **Resource Group**, and **Region** in which to deploy the Application, then click **Next**.<br/><img src={useBaseUrl('img/send-data/azure-metrics-source/basics.png')} alt="basics" style={{border:'1px solid gray'}} width="500"/>
+1. Under the **App Registration** tab:<br/><img src={useBaseUrl('img/send-data/azure-metrics-source/app-registration.png')} alt="app-registration" style={{border:'1px solid gray'}} width="500"/> 
+    1. Select service principal type as **Create new** to create a new app.
+    1. Click **Change selection**, to enter the name and select **Accounts in any organizational directory (Any Microsoft Entra ID tenant - Multitenant)** as supported account types. Click **Register** to register the application.<br/><img src={useBaseUrl('img/send-data/azure-metrics-source/register.png')} alt="register" style={{border:'1px solid gray'}} width="400"/> 
+1. In the re-directed **Certificates & secrets** page, under **Client sectrets** tab, click **+ New client secret**.<br/><img src={useBaseUrl('img/send-data/azure-metrics-source/new-client-secret.png')} alt="new-client-secret" style={{border:'1px solid gray'}} width="600"/> 
+        1. **Description**. Enter the description of your choice.
+        1. **Expires**. From the dropdown select the expiry time period to the secret value. 
+1. Navigate to **Overview**, copy and save the **Application (client) ID** and **Directory (tenant) ID**.<br/><img src={useBaseUrl('img/send-data/azure-metrics-source/overview.png')} alt="overview" style={{border:'1px solid gray'}} width="600"/>  
+1. Under **Manage**, navigate to **Certificates & secrets** page, copy and save the **Value**.<br/><img src={useBaseUrl('img/send-data/azure-metrics-source/client-secrets.png')} alt="client-secrets" style={{border:'1px solid gray'}} width="600"/>   
```

Comment:
Nit: Should we ask customer to copy and save client secrets first before they navigate to Overview? Customer can save one route to the secrets page. @samjsong

Reply:
Should we also mention here that client secret will be later used in source configuration?

Reply:
> Should we also mention here that client secret will be later used in source configuration?

I think we could do this, but I don't feel strongly about it either way since we mention it on Line 27 that the purpose of these steps is to get these three values.

> Should we ask customer to copy and save client secrets first before they navigate to Overview? Customer can save one route to the secrets page.

This makes sense to me. @JV0812 Can we combine step 4 and step 6, so that customer can just create the secret (step 4) and then immediately save it (step 6)?

Reply:
updated!